src/co_nutate.jl

```diff
@@ -62,11 +62,11 @@ Example 23a in Meeus: On 2028 Nov 13.19 TD the mean position of Theta
 Persei is 2h 46m 11.331s 49d 20' 54.54''. Determine the shift in
 position due to the Earth's nutation.
 
-```julia
+```jldoctest
 julia> jd = jdcnv(2028,11,13,4,56)
 2.4620887055555554e6
 
-julia> co_nutate(jd,ten(2,46,11.331)*15,ten(49,20,54.54))
+julia> co_nutate(jd,ten(2,46,11.331) * 15,ten(49,20,54.54))
 (0.006058053578186673, 0.002650870610381162, 0.40904016038217567,
```

Comment:
For this test I get
```
(0.006058053578186673, 0.0026508706103953728, 0.40904016038217555, 14.85938942789647, 2.703809037235058)
```
as result.  Please, don't break line if it isn't broken in the REPL output.